### ISSUE TITLE
GLIBC error on Ubuntu 20.04

### ISSUE THREAD

**Issue by iainelder** [Versent/saml2aws#1026]
I get the same error as reported in #1010 when trying to execute the installed `saml2aws_2.36.6_linux_amd64.tar.gz` on Ubuntu 20.04.

```console
$ saml2aws --version
saml2aws: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by saml2aws)
saml2aws: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by saml2aws)
saml2aws: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by saml2aws)
```

I use GitHub actions to test my installation script. You can see the complete installation log and error in the latest [CI run](https://github.com/iainelder/dotfiles/actions/runs/4631353920/jobs/8194111171).

Does saml2aws still support Ubuntu 20.04?

**Comment by chris-reeves** [comment]
I'd suggest closing this as a duplicate of the other issue that you linked to, as these are the same issue.

**Comment by iainelder** [comment]
@chris-reeves commented on #1010:

> This is a result of the merge of https://github.com/Versent/saml2aws/pull/951. That PR forces CGO_ENABLED=1, which introduces a dependency on libc6. As this is built on ubuntu-latest (currently 22.04) the minimum version of libc supported is 2.32, which is greater than that available in Ubuntu 20.04 and Debian stable.

I take that to mean that Ubuntu 20.04 is no longer supported, which answers my question. Thanks!

**Comment by chris-reeves** [comment]
> I take that to mean that Ubuntu 20.04 is no longer supported, which answers my question. Thanks!

Effectively, yes, at the moment. But I don't believe that was intentional and ultimately I would like to see #1010 resolved.
